smokes/e2e/pages/builder.coffee

```diff
@@ -62,4 +62,10 @@ class builderPage
     getRebuildButton: ->
         return element(By.buttonText('Rebuild'))
 
+    checkBuilder: (builderLink) ->
```

Comment:
I would call this method checkBuilderURL()

then no need to pass builderLink, it is already set when contructing the builderPage